ml_tools/eolearn/ml_tools/train_test_split.py

```diff
@@ -1,17 +1,33 @@
 """
 Tasks used for train set preparation.
 """
+from enum import Enum
 import numpy as np
 
 from eolearn.core import EOTask
 
 
+class TrainTestSplitType(Enum):
+    """ An enum defining TrainTestSplitTask's methods of splitting the data into subsets
+    """
+    PER_PIXEL = 'per_pixel'
+    PER_CLASS = 'per_class'
+    PER_VALUE = 'per_value'
+
+
 class TrainTestSplitTask(EOTask):
     """ Randomly assign each or group of pixels to a train, test, and/or validation sets.
 
     The group of pixels is defined by an input feature (i.e. MASK_TIMELESS with polygon ids, or connected component id,
-    or similar that groups pixels with similar properties). By default pixels get assigned to a subset 'per_class'. If
-    split_type is set to 'random', then pixels get randomly assigned to a subset, regardless of their class.
+    or similar that groups pixels with similar properties). By default pixels get assigned to a subset
+    :attr:`PER_CLASS<eolearn.ml_tools.train_test_split.TrainTestSplitType.PER_CLASS>`, where pixels of the same class
+    will be assigned to the same subset. On the other hand if split_type is set to
+    :attr:`PER_PIXEL<eolearn.ml_tools.train_test_split.TrainTestSplitType.PER_PIXEL>`, each pixel gets randomly
+    assigned to a subset regardless of its class. The third option is splitting
+    :attr:`PER_VALUE<eolearn.ml_tools.train_test_split.TrainTestSplitType.PER_VALUE>`, where each class is assigned to
+    a certain subset consistently across eopatches. In other words, if a polyogn lies on multiple eopatches, it will
```

Comment:
What about 'pixel' instead of 'value'? It's used in the text multiple times already. Either 'value' or 'pixel', it would maybe be good to be consistent and use just one of them.

Reply:
Refactored the whole docstring more generally.